Shared/WriteFreely_MultiPlatformApp.swift

```diff
@@ -170,7 +170,7 @@ struct WriteFreely_MultiPlatformApp: App {
         let managedPost = model.editor.generateNewLocalPost(withFont: model.preferences.font)
         withAnimation {
             // Set it as the selectedPost
-            DispatchQueue.main.asyncAfter(deadline: .now()) {
+            DispatchQueue.main.asyncAfter(deadline: .now() + 0.25) {
```

Comment:
Similarly, wait for the value of `model.selectedPost` to settle before setting it to a new draft.